model/implementation/DeliveryExecutionStateService.php

```diff
@@ -57,6 +57,9 @@ class DeliveryExecutionStateService extends AbstractStateService implements \oat
     const OPTION_CANCELLATION_DELAY = 'cancellation_delay';
     const OPTION_TIME_HANDLING = 'time_handling';
 
+    const TIME_HANDLING_EXTRA_TIME = 'extra_time';
```

Comment:
I'm just wondering : what if we rename this variable https://github.com/oat-sa/extension-tao-proctoring/blob/8b98ae91a1ce06954d9456ad34ad1cabaee3784a/model/monitorCache/DeliveryMonitoringService.php#L55, then this one should be renamed as well? then I prefer only 1 constant

Reply:
The constant you've linked is concerning a key to use when storing data in `delivery_monitoring` table. While the new constant in this file is for the values valid for option `OPTION_TIME_HANDLING`